src/pybind/mgr/cephadm/module.py

```diff
@@ -2389,6 +2389,16 @@ def upgrade_check(self, image: str, version: str) -> str:
             raise OrchestratorError('must specify either image or version')
 
         image_info = CephadmServe(self)._get_container_image_info(target_name)
+
+        target_version = image_info.ceph_version
```

Comment:
Can we rename `target_version`? Took a while to see the difference between `version` and `target_version`

Reply:
changed it to `ceph_image_version`